deploy_baseos_to_master1.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# master1에 ansible 디렉토리 복사 및 베이스라인 자동화
+
+set -e
+
+
+# 변수: master1 접속 정보
+MASTER1_IP="$1"
+SSH_USER="chanyong"  # GCP에서 권장하는 계정명
+SSH_KEY="$2"      # ex) ~/.ssh/GCPKEY
+
+# 1. ansible 디렉토리 복사 (ansible.cfg 포함)
+scp -i "$SSH_KEY" -r ../../../../infra/ansible "$SSH_USER@$MASTER1_IP:~/ansible"
```

Comment:
아래 상대 경로를 절대 경로로 바꿔 주세요.
```suggestion
scp -i "$SSH_KEY" -r /absolute/path/infra/ansible "$SSH_USER@$MASTER1_IP:~/ansible"
```